rascal-lsp/src/main/rascal/lang/rascal/tests/rename/TestUtils.rsc

```diff
@@ -203,7 +203,7 @@ bool testRename(str stmtsStr, int cursorAtOldNameOccurrence = 0, str oldName = "
 public PathConfig getTestPathConfig(loc testDir) {
     return pathConfig(
         bin=testDir + "bin",
-        libs=[|lib://rascal|],
+        libs=[|std://rascal|],
```

Comment:
That's not right. Std is only a source path for the interpreter. If we need the class files of the Rascal project here, for the java part of the stdlib, then use the functions to get a loc for the current Rascal project. I forget their name.

Reply:
Srcs must contain std:// since the merge because it is not added by itself. PathConfig adds it and all other code that constructs pathConfig must do it too.

Reply:
I think it's not needed to add rascal.jar to the libs as the interpreter already runs in that jar and so the lib classes are loaded from there.

Reply:
I think this is the result of an automatic search & replace

Reply:
Yes it certainly looks like that. Let's replace by the gerCurrentRascl for now